test/fixtures/v5/continue/uturn_exit.json

```diff
@@ -0,0 +1,22 @@
+{
+    "step": {
+        "maneuver": {
+            "type": "continue",
+            "modifier": "uturn"
+        },
+        "name": "Way Name",
+        "exits": "4A,4B"
+    },
+    "instructions": {
+        "de": "180°-Wendung auf Way Name",
+        "en": "Make a U-turn onto Way Name",
```

Comment:
Exits only make sense in the off-ramp case as far as I know. I would be surprised if there are any tags used in other context, but I think we should focus on the highway/ramp use case first.